FunctionalTests/Microsoft.Bot.Builder.FunctionalTests/WebexClientTest.cs

```diff
@@ -100,41 +100,47 @@ private async Task RefreshAccessToken()
 
             var httpResponse = await client.SendAsync(request);
             var response = httpResponse.Content.ReadAsStringAsync().Result;
-            var kvPairs = JsonConvert.DeserializeObject<Dictionary<string, string>>(response);
-
-            _userAccessToken = kvPairs["access_token"];
+            try
+            {
+                var kvPairs = JsonConvert.DeserializeObject<Dictionary<string, string>>(response);
+                _userAccessToken = kvPairs["access_token"];
+            }
+            catch
+            {
+                throw new Exception($"RefreshAccessToken() failed: response = {response}");
+            }
```

Comment:
Why throw a new exception? If the operation throws, the test will fail with the error needed. If you want to wrap the exception, you can add the original exception as an inner exception so the information is not lost.

Reply:
Thanks for the suggestion. Inner exception now captured.